### ISSUE TITLE
Add publiccode.yml

### ISSUE THREAD

**Issue by nemobis** [phpList/phplist3#544]
Would you be interested in a patch to add a metadata file like https://github.com/globaleaks/GlobaLeaks/blob/master/publiccode.yml ? It's necessary to be included in https://developers.italia.it/en/software/ , the list of free software which the Italian state will heavily advertise to the public sector in Italy.

**Comment by samtuke** [comment]
Yes, sounds good. Some of the data needa to be updated for each release, but updating the release script used to roll releases should be easy.
